app/serializers/publication-flow.js

```diff
@@ -0,0 +1,27 @@
+import ApplicationSerializer from './application';
+
+const SKIP_SERIALIZED = ['publicationStatusChange'];
+
+export default class PublicationFlowSerializer extends ApplicationSerializer {
+  serializeBelongsTo(snapshot, json, relationship) {
+    const key = relationship.key;
+    if (!SKIP_SERIALIZED.includes(key)) {
+      super.serializeBelongsTo(snapshot, json, relationship);
+    }
+  }
+
+  serializeHasMany(snapshot, json, relationship) {
+    const key = relationship.key;
+    if (!SKIP_SERIALIZED.includes(key)) {
+      super.serializeHasMany(snapshot, json, relationship);
+    }
+  }
+
+  shouldSerializeHasMany(snapshot, key, relationshipType) {
+    if (key === 'mandatees') {
+      return true;
+    } else {
+      return super.shouldSerializeHasMany(snapshot, key, relationshipType);
+    }
+  }
+}
```

Comment:
Don't fully get this.
I think all hasMany's that get loaded get saved unless you skip them.
Why would it be different in this case but not other hasMany's that do get serialized?

Reply:
According to [the docs](https://api.emberjs.com/ember-data/4.9/classes/JSONAPISerializer/methods/shouldSerializeHasMany?anchor=shouldSerializeHasMany), it's indeed not necessary for `many-to-many`.

> By default only many-to-many and many-to-none relationships are serialized.

Reply:
I'm removing this `shouldSerializeHasMany` here since it's not necessary, afterwards the PR should be okay change-wise (at least AFAIK)